<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cell_Main</class>
 <widget class="QMainWindow" name="Cell_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1377</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图书管理系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton[name=&quot;btn&quot;]{
	border-radius:4px;
	background:#409eff;
	color:white;
	font-size:14px;
	min-height:28px;
	min-width:85px;
	max-width:85px;
	padding:4px;
}
QPushButton[name=&quot;btn&quot; ]:pressed{
	background:#409edd;
}

QLineEdit[name=&quot;le&quot;]{
	border-radius:4px;
	min-height:25px;
	min-width:200px;
	max-width:200px;
	border:1px solid gray;
}

QLabel[name=&quot;title&quot;]{
	font:18px '黑体';color:blue;
}
QLabel[name=&quot;lb&quot;]{
	font-size:14px '黑体';
}
QWidget[name=&quot;bg&quot;]{
	background-color:white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#tool{background-color:rgb(84,92,100);}

QWidget#tool QToolButton{
	border:none;
	background-color:rgb(84,92,100);
	color:white;
	min-height:55px;
	font:18px &quot;黑体&quot;;
	padding-left:12px;
}

QWidget#tool QToolButton::checked{
	background-color:rgb(67,74,80);	
}
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="tool" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QToolButton" name="btn_user">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>用户管理</string>
         </property>
         <property name="icon">
          <iconset resource="../res/res.qrc">
           <normaloff>:/tool1.svg</normaloff>:/tool1.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="btn_book">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>图书管理</string>
         </property>
         <property name="icon">
          <iconset resource="../res/res.qrc">
           <normaloff>:/tool1.svg</normaloff>:/tool1.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QToolButton" name="btn_record">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>借阅记录</string>
         </property>
         <property name="icon">
          <iconset resource="../res/res.qrc">
           <normaloff>:/tool1.svg</normaloff>:/tool1.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QToolButton" name="btn_help">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>使用帮助</string>
         </property>
         <property name="icon">
          <iconset resource="../res/res.qrc">
           <normaloff>:/tool1.svg</normaloff>:/tool1.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
